csrc/mmdeploy/preprocess/transform/load.cpp

```diff
@@ -52,6 +53,13 @@ Result<Value> PrepareImageImpl::Process(const Value& input) {
 
   SetTransformData(output, "img", std::move(tensor));
 
+  // trace static info & runtime args
+  Tracer tracer;
```

Comment:
Can we add `output["__tracer__"]` in `Compose::Compose(...)`?
So that each transform op won't be bothered by `if (output.contains("__tracer__"))`

Reply:
if (output.contains("__tracer__")) is for mmdeploy_tests.

Only load will add __tracer__ to the output. If we only test crop like (tests/test_csrc/preprocess/test_crop.cpp), it will failed because there is no __tracer__ in output.